packages/atomic-angular/README.md

```diff
@@ -36,6 +36,43 @@ export class AppModule {}
 
 Once this is done, you will be able to reference all atomic components inside that module.
 
+## Initializing the search interface
+
+You can initialize the search interface at any time you desire in the lifecycle of your application. One such suitable lifecyle hook is the `AfterViewInit`,
+
+For example, using the above `AppModule` as a starting point, which bootstrap `AppComponnent`:
```

Comment:
I don't really see what you mean here: "which bootstrap `AppComponnent`" ?

Reply:
If you look at the full README, in the `AppModule` code, there's a `bootstrap` configuration